ramls/defaultRules/defaultRules.json

```diff
@@ -0,0 +1,101 @@
+{
+  "rules": [
+    {
+      "ruleId": "5105b55a-b9a3-4f76-9402-a5243ea63c95",
+      "name": "password_length",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "^.{8,}$",
+      "description": "The password length must be minimum 8 digits",
+      "orderNo": 0,
+      "errMessageId": "password.length.invalid"
+    },
+    {
+      "ruleId": "dc653de8-f0df-48ab-9630-13aacfe8e8f4",
+      "name": "alphabetical_letters",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "(?=.*[a-z])(?=.*[A-Z]).+",
+      "description": "The password must contains both upper and lower case letters",
```

Comment:
Change `contains` to `contain`, please.  Check similar cases.

Reply:
changed, thank you